R/Model-methods.R

```diff
@@ -285,12 +286,16 @@ setMethod(
     samples = "Samples"
   ),
   definition = function(x, model, samples) {
+    assert_probabilities(x)
     assert_subset(c("alpha0", "alpha1"), names(samples@data))
+    nsamples <- sampleSize(samples@options)
+    if (nsamples > 1L) {
+      assert_length(x, len = c(1L, nsamples))
+    }
```

Comment:
could write in one line to save some real estate
```suggestion
    if (nsamples > 1L) assert_length(x, len = c(1L, nsamples))
```